frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -0,0 +1,185 @@
+    <Stack hasGutter>
+      <StackItem>
+        <LogsTabStatus
+          loaded={loaded}
+          error={error}
+          refresh={refreshLogs}
+          onDownload={onDownload}
+          canDownload={canDownload}
+        />
+      </StackItem>
+      <StackItem>
+        <Split hasGutter isWrappable>
+          <SplitItem>
+            <Split hasGutter>
+              <SplitItem>
+                <Bullseye>
+                  <TextContent>
+                    <TextListItem component={TextListItemVariants.dt}>Container</TextListItem>
+                  </TextContent>
+                </Bullseye>
+              </SplitItem>
+              <SplitItem>
+                <SimpleDropdownSelect
+                  isDisabled={podContainers.length === 0}
+                  options={podContainers.map((container) => ({
+                    key: container.name,
+                    label: container.name,
+                  }))}
+                  value={selectedContainer?.name ?? ''}
+                  placeholder="Select container..."
+                  onChange={(v) => {
+                    setSelectedContainer(podContainers.find((c) => c.name === v) ?? null);
+                  }}
+                  width={200}
+                />
+              </SplitItem>
+            </Split>
+          </SplitItem>
+          <SplitItem>
+            <Button
+              variant={!logsLoaded ? 'plain' : isPaused ? 'plain' : 'link'}
+              onClick={() => setIsPaused(!isPaused)}
+              isDisabled={!!error}
+            >
+              {error ? (
+                'Error loading logs'
+              ) : !logsLoaded ? (
+                <>
+                  <Spinner size="sm" /> Loading log
+                </>
+              ) : isPaused ? (
+                <>
+                  <PlayIcon /> Resume refreshing
+                </>
+              ) : (
+                <>
+                  <PauseIcon /> Pause refreshing
+                </>
+              )}
+            </Button>
+          </SplitItem>
+          <SplitItem isFilled style={{ textAlign: 'right' }}>
+            <Tooltip position="top" content={<div>Download currently selected container</div>}>
+              <Button
+                onClick={onDownload}
+                variant="link"
+                aria-label="Download current logs"
+                icon={<DownloadIcon />}
+                isDisabled={!canDownload}
+              >
+                Download
+              </Button>
+            </Tooltip>
+          </SplitItem>
+        </Split>
+      </StackItem>
+      <StackItem isFilled>
+        <DashboardCodeEditor
+          onEditorDidMount={onMount}
+          code={data}
+          isReadOnly
+          options={editorOptions}
+        />
+        {isPaused && (
+          <Button onClick={() => setIsPaused(false)} isBlock icon={<OutlinedPlayCircleIcon />}>
+            Resume refreshing log
+          </Button>
+        )}
+      </StackItem>
+    </Stack>
```

Comment:
Lots of extra whitespace below the editor when this button isn't rendered:
![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/b841ce50-95a4-4b83-a1b1-37a908fac7a7)

Reply:
Is the log viewer currently set to a fixed height that matches the height of the side drawer? My apologies for the question, as I'm not familiar with running the code on my end.

Reply:
The log viewer is currently set to expand and consume all remaining available height in the side drawer. However there is excess white space at the bottom as seen in the screenshot. When the resume button appears, the spacing looks correct. It's as if the space is reserved for the resume button at all times whereas when the resume button is hidden, I expect the editor to consume the available space.

Reply:
That makes sense. Thanks for the explanation!